### ISSUE TITLE
Accessibility sections in documentation

### ISSUE THREAD

**Issue by cjolif** [ibm-js/deliteful#471]
I have found several issues with accessibility sections in doc:

* [x] Button, it says "same accessibility as standard button". We do provide some theming so we need at least to test if it is high contrast ok. In general I would like even if we rely on browser to see the 3 sections, keyboard, visual formatting, & screen reader with the status
* [x] LinearLayout, same as above, for example JAWS announcing the children in the correct order? This needs to be tested and put in a table
* [x] ResponsiveContainer, same as above, for example JAWS announcing the children in the correct order? This needs to be tested and put in a table
* [x] Select is not following the typical table we have to report status wrt to the 3 accessibility items.
* [x] SidePane, How can I open the side pane with keyboard? Looks like at least this is not keyboard accessible? Same as others, table with status is needed.
* [x] Slider, missing JAWS in the table. If not tested it should explictly state it.
* [x] StarRating, not using the standard table to report accessibiltiy. Not mentionning visual formatting.
* [x] SwapView, link to ViewStack as ViewStack seems to provide accessibility?
* [ ] ViewIndicator, link ViewStack?
* [ ] ViewStack, results of testing? What about the table with the 3 items?

**Comment by cjolif** [comment]
@AdrianVasiliu can you handle Select and check it once done? @dmandrioli as you get most of the others, can you handle the rest of it? (please contacts directly initial owners when you have questions)

**Comment by wkeese** [comment]
It sounds like you're confusing testing and documentation.   When you say things like:

> Button, it says "same accessibility as standard button". We do provide some theming so we need at least to test if it is high contrast ok.

Sure, you need to test high contrast, but what's the point of documenting that you tested it?

**Comment by cjolif** [comment]
> Sure, you need to test high contrast, but what's the point of documenting that you tested it?

Because in some cases we have exceptions of some parts being not working..., so the best way to know the status of testing is to document it explicitly.

**Comment by AdrianVasiliu** [comment]
> @AdrianVasiliu can you handle Select

Done via cd7038e87b2066eb4a1ddaf6ac10ecec66972fef.

**Comment by dmandrioli** [comment]
>as ViewStack seems to provide accessibility?

That's not true, the doc of SwipeView has been fixed accordingly in https://github.com/dmandrioli/deliteful/commit/494a6411acb02379c6c90e7f1595f1cd7e1a4fce
